icu_benchmarks/models/wrappers.py

```
@@ -459,17 +471,21 @@ def predict(self, features):
 
     def log_metrics(self, label, pred, metric_type):
         """Log metrics to the PL logs."""
-
+        if "Confusion_Matrix" in self.metrics.keys():
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)